packages/haul-core/assets/debuggerWorker.js

```diff
@@ -117,16 +117,18 @@ class DebuggerWorker {
 
     this.shouldQueueMessages = true;
 
+    let error;
     try {
       importScripts(message.url)
     } catch (e) {
+      error = e;
       if (self.ErrorUtils) {
         self.ErrorUtils.reportFatalError(e);
       } else {
         console.error(e);
       }
     } finally {
-      sendReply(null, null);
+      sendReply(null, error);
```

Comment:
I guess it doesn't matter if error is `null` or `undefined`?

Reply:
It does, it has to have some value or be `undefined`. `null` is incorrectly handled on iOS.